webapp/src/shared/components/Game/hooks/useGameAnimation.tsx

```diff
@@ -5,22 +5,15 @@ import {
   BRICK_WIDTH,
   BRICK_HEIGHT,
   BALL_RADIUS,
-} from './constants';
-import { GameBall, GamePaddle } from './types';
-import { getBallPos, getPaddlePos, initialBallState } from './physics';
+  CANVAS_HEIGHT,
+} from '../constants';
+import { GameBall, GamePaddle, GameState } from '../models';
 
-const useGameAnimation = (
-  ballRef: React.MutableRefObject<GameBall>,
-  paddleRef: React.MutableRefObject<GamePaddle>,
-) => {
+const useGameAnimation = () => {
   const deltaTimeRef = React.useRef<number>(0);
 
   const drawBall = React.useCallback(
-    (context: CanvasRenderingContext2D) => {
-      const ball = ballRef.current;
-      const paddle = paddleRef.current;
-
-      ballRef.current = getBallPos(ball, paddle, deltaTimeRef.current);
+    (context: CanvasRenderingContext2D, ball: GameBall, paddle: GamePaddle) => {
```

Comment:
`paddle: GamePaddle` is unused here and now. Why do you think the callback to draw the Ball needs the paddle? Is it also scaffolding or just a mistake?

Reply:
It was a leftover, removed!